medusa/system/schedulers.py

```diff
@@ -0,0 +1,139 @@
+# coding=utf-8
+"""Helper functions to get info about running schedulers."""
+from __future__ import unicode_literals
+
+from medusa import app
+from medusa.generic_queue import QueuePriorities
+from medusa.helpers.utils import timedelta_in_milliseconds
+from medusa.show_queue import ShowQueueActions
+
+
+all_schedulers = [
+    ('dailySearch', 'Daily Search', 'daily_search_scheduler'),
+    ('backlog', 'Backlog', 'backlog_search_scheduler'),
+    ('showUpdate', 'Show Update', 'show_update_scheduler'),
+    ('versionCheck', 'Version Check', 'version_check_scheduler'),
+    ('showQueue', 'Show Queue', 'show_queue_scheduler'),
+    ('searchQueue', 'Search Queue', 'search_queue_scheduler'),
+    ('properFinder', 'Proper Finder', 'proper_finder_scheduler'),
+    ('postProcess', 'Post Process', 'auto_post_processor_scheduler'),
```

Comment:
Note to self - #6810 affects this line
```suggestion
    ('postProcess', 'Post Process', 'post_processor_scheduler'),
```